gasaerexch/plot_gasaerexch_timestep.py

```diff
@@ -22,7 +22,36 @@ def __init__(self):
                                                   data.output.qso4_gas_rel, \
                                                   data.output.qsoa_gas_rel
         self.pmid, self.temp, self.dtsubstep = data.input.pmid, data.input.temp,  data.input.dtsubstep
-        
+ 
+        dtrange = set()
+        for time in data.input.dtsubstep:
+            if time not in dtrange:
```

Comment:
A `set` is an ordered set of unique elements, so you don't need to check whether something is in it before calling `add`. For example, if I have a set `A = {1, 2, 3}`, and I try to add `3` to it, nothing happens. The set maintains the uniqueness of all the elements.

Reply:
(That `{1, 2, 3}` syntax is the mathematical notation for a set, not Python. 🙂 )

Reply:
> Looks good. I just left a couple of comments about sets: you don't need to check for uniqueness yourself--the set does that for you.

Thanks for reviewing. I will edit these parts of the code.